client/src/Components/Search/SearchPageRecentTransactions.jsx

```diff
@@ -0,0 +1,44 @@
+import {useEffect, useState} from "react"
+import "../Search/SearchPage.css"
+import { FaLongArrowAltRight, FaLongArrowAltLeft } from "react-icons/fa";
+function RecentTransactions(props) {
+  const [cryptoInfo, setCryptoInfo] = useState([])
+  const cryptoList= props.cryptoInfo;
+
+useEffect(() => {
+setCryptoInfo(cryptoList)
+},[cryptoList])
+  return (
+    <div>
+      <div
+        className="artboard phone-2"
+        style={{
+          backgroundColor: "aliceblue",
+          height: 400,
+          marginLeft: 50,
+          marginTop: 450,
+          borderRadius: 10,
+          overflowX: "hidden",
+          overflowY: "auto",
+        }}
+      >
+        {cryptoInfo.length === undefined && cryptoInfo.tickers.map((crypto) => {
+        return <div>

```

Comment:
could you change this to `<div key={crypto.id}>` ?
it seems like react requires you to have a key attribute